### ISSUE TITLE
Sort "List Users" endpoint?

### ISSUE THREAD

**Issue by tejasmanohar** [intercom/intercom-go#47]
How do you sort the "List Users" data?

**Comment by choran** [comment]
At the moment you cannot sort data via the list users functionality. i..e you cannot set a sort field at the moment. It this what you mean? i.e. how do you sort data similar to the API for list users?

**Comment by tejasmanohar** [comment]
Yeah, meant setting a sort field. Alright, I'll send a PR. It seems you can here- https://developers.intercom.io/reference

**Comment by choran** [comment]
Hi @tejasmanohar 
Thanks for submitting the PR and sorry for not getting to it yet.
Should be able to get to it this week
Cheers
Cathal

**Comment by tejasmanohar** [comment]
@choran Ooh, there are merge conflicts now (almost a yr later :P). I'm no longer using this library and don't remember the context of this well so if these changes are valuable, it may be best to incorporate them outside of my PR or amend it on your end. Sorry!

**Comment by choran** [comment]
Hi @tejasmanohar,
Yeah, that is fair enough. We are making an effort to get to these regularly now so that they do not remain inactive for so long. For now I will make this a work needed and close the PR for now.
Thanks for submitting that and apologies for not getting to it before

**Comment by tejasmanohar** [comment]
All good! Thanks

**Comment by choran** [comment]
np, I will leave this issue open in the meantime since, as noted, it is functionality that would be useful in the sdk
